rasa/core/channels/slack.py

```diff
@@ -293,11 +307,15 @@ def _get_interactive_response(action: Dict) -> Optional[Text]:
 
             return response.text(None, status=201, headers={"X-Slack-No-Retry": 1})
 
+        if metadata is not None:
```

Comment:
this can't be `None` in your implementation, so you don't have to check for it.

Reply:
This is required for typing. As process_message, contains metadata as an option.